roles/confluent.kafka/templates/server_kerberos_ssl.self_signed.properties.j2

```diff
@@ -2,7 +2,7 @@
 
 {% include './includes/base_server.properties.j2' %}
 
-listeners=SASL_SSL://:{{broker.config.port}}
+listeners=SASL_SSL://:{{kafka_port}}
```

Comment:
Recommend leaving in 'broker' as kafka is not a component but the platform name.